### PR TITLE
Update try-runtime-cli

### PR DESCRIPTION
**Pull request by ggwpez**
- Update try-runtime-cli for MBM testing that was added https://github.com/paritytech/try-runtime-cli/pull/90
- Fix KAH spec version that was incorrectly bumped
- Disable MBM testing for Encointer runtime until https://github.com/paritytech/try-runtime-cli/issues/101 is fixed (single-block-migrations are still checked).

- [x] Does not require a CHANGELOG entry

### CODE REVIEW

File: system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs [hunk 1]
```diff
@@ -137,7 +137,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("statemine"),
 	impl_name: create_runtime_str!("statemine"),
 	authoring_version: 1,
-	spec_version: 1_003_000,
+	spec_version: 1_003_003,
```
Comment (ggwpez) [inline system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:140]: We forgot to bump this and only bumped the feature-gated version above...

**Review by acatangiu — APPROVED** [review]

**Review by seadanda — APPROVED** [review]

**Review by ordian — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Comment by ggwpez** [comment]
> I think it needs another review, my review isn't counted because I'm a rank 1 pleb

Hm, its weird that the bot is not responding though. It should print some error.

**Comment by seadanda** [comment]
https://github.com/polkadot-fellows/runtimes/pull/463/checks?check_run_id=31685484077
Is showing that 1 review rank 3+ is missing

File: .github/workflows/check-migrations.yml [hunk 2]
```diff
@@ -82,16 +84,51 @@ jobs:
...
+        run: rustup component add rust-src
+        shell: bash
+
+      - name: Run ${{ matrix.runtime.name }} Runtime Checks
+        #uses: "paritytech/try-runtime-gha@v0.2.0"
+        env:
+          EXTRA_FLAGS: ${{ env.EXTRA_FLAGS }}
+          CHECKS: ${{ env.CHECKS }}
+        run: |
+          cargo install -q --git https://github.com/liamaharon/try-runtime-cli --branch liam-runtime-upgrade-mbms
```
Comment (ggwpez) [inline .github/workflows/check-migrations.yml:109]:
```suggestion
          cargo install -q --git https://github.com/liamaharon/try-runtime-cli --branch liam-runtime-upgrade-mbms
```

[the remark this answers is not in the document]
Comment (ggwpez) [reply .github/workflows/check-migrations.yml]: Oh, its merged since. Fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix missing bump of the Kusama-Asset-Hub spec version to 1.3.3 ([polkadot-fellows/runtimes#463](https://github.com/polkadot-fellows/runtimes/pull/463))
+
```
Comment (bkchr) [inline CHANGELOG.md:12]:
```suggestion
```

Not convinced we need a changelog entry for this.

File: .github/workflows/check-migrations.yml [hunk 4]
```diff
@@ -82,16 +84,51 @@ jobs:
...
+        run: rustup component add rust-src
+        shell: bash
+
+      - name: Run ${{ matrix.runtime.name }} Runtime Checks
+        #uses: "paritytech/try-runtime-gha@v0.2.0"
+        env:
+          EXTRA_FLAGS: ${{ env.EXTRA_FLAGS }}
+          CHECKS: ${{ env.CHECKS }}
+        run: |
+          cargo install -q --git https://github.com/paritytech/try-runtime-cli
```
Comment (bkchr) [inline .github/workflows/check-migrations.yml:109]: We should put there some `--rev` to install always the same version.

Comment (bkchr) [reply .github/workflows/check-migrations.yml:109]: And also `--locked`.

**Review by bkchr — APPROVED** [review]
Some last nitpicks and then this is good to go

File: .github/workflows/check-migrations.yml (hunk 4 above)
Comment (ggwpez) [inline .github/workflows/check-migrations.yml:109]:
```suggestion
          cargo install -q --git https://github.com/paritytech/try-runtime-cli --tag v0.8.0 --locked && try-runtime --version
```

**Pull request merged**